src/components/common/Dot/Dots.module.scss

```diff
@@ -0,0 +1,77 @@
+@import "../../../styles/mediaQueriesMixins.scss";
+@import "../../../styles/settings.scss";
+
+.dots {
+    position: absolute;
+    right: 0;
+    top: 0;
+
+    @include for-tablet {
+        position: relative;
+        margin: 0 auto;
+    }
+
+    @include for-phone {
+        top: 60px;
+    }
+
+    ul {
+        margin: 0;
+        padding: 0 0 0 1rem;
+        list-style: none;
+        background-color: $newFurniture-dots-ul-bg;
+        transform: translateY(15px);
+
+        li {
+            display: inline-block;
+            margin-left: 0.5rem;
+
+            a {
+                display: block;
+                width: 13px;
+                height: 13px;
+                border-radius: 6px;
+                background-color: $newFurniture-dots-bg;
+                font-size: 0;
+
+                &.active,
+                &:hover {
+                    @include bg-hover-animation;
+                    background-color: $newFurniture-active-dots-bg;
+                }
+            }
+        }
+    }
+}
+
+.hotDealsDots {
```

Comment:
Komponent generyczny powinien być idealnie napisany w taki sposób, aby go móc skopiować i wkleić do jakieś w ogóle innej aplikacji i nadal tam będzie działać. Tutaj w komponencie generycznym dodałeś mu bardzo specyficzny przypadek jakieś `hotDeals` i ktoś kto chce użyć `Dots` musi zdawać sobie sprawę co to jest `hotDeals` więc jak już, to takie coś powinno mieć uniwersalną nazwę opisującą jak to wpłynie na zachowanie tego komponentu. np. button active, jakieś wersjie small big itp.

Tutaj mamy specyficzne zachowanie w określonym miejscu na stronie i chodzi głównie o pozycjonowanie, to powinno być zrobione w inny sposób. Dots zawsze powinny wyświetlać się "normalnie". Jeśli jakiś komponent potrzebuje wyświetlić kropki w jakiś specyficzny sposób (absolutnie) to w tym komponencie należy zrobić np. `div` i jego ustawić gdzie trzeba a w środku wyświetlić `Dots`